client/components/theme/index.jsx

```diff
@@ -123,7 +123,7 @@ export class Theme extends Component {
 					aria-label={ this.props.theme.name }
 					title={ this.props.theme.description }
 					className="theme__active-focus"
-					href={ this.props.screenshotClickUrl || '#' }
+					href={ this.props.screenshotClickUrl || 'javascript:;' }
```

Comment:
Maybe add a comment above this line explaining this choice, so others won't be tempted to replace with `#` or remove altogether.